pkg/apis/serving/v1beta1/predictor.go

```diff
@@ -67,6 +67,9 @@ type PredictorSpec struct {
 	PodSpec `json:",inline"`
 	// Component extension defines the deployment configurations for a predictor
 	ComponentExtensionSpec `json:",inline"`
+
+	// Enables collocation of transformer with predictor
+	Processors []corev1.Container `json:"processors,omitempty"`
```

Comment:
How do we plan to merge the processor with the multiple containers defined on serving runtime ?

Reply:
As discussed with @sivanantha321 `containers` are already defined on PodSpec, we can use that as the additional processor containers to the model container (kserve-container)